mars/services/task/analyzer/analyzer.py

```diff
@@ -303,6 +303,12 @@ def gen_subtask_graph(self) -> SubtaskGraph:
             "Assigned %s start chunks for task %s", len(start_ops), self._task.task_id
         )
 
+        # assign expect workers for successors
```

Comment:
"assign expect workers for those specified with expect_worker" seems clearer.

Reply:
good idea